xblockutils/deprecation/warn.py

```diff
@@ -0,0 +1,34 @@
+"""
+Utilities for warning about the use of deprecated package
+
+See https://github.com/openedx/xblock-utils/issues/197 for details.
+"""
+
+import warnings
+
+
+class DeprecatedPackageWarning(DeprecationWarning):
+    """
+    A warning that a deprecated package is being used.
+    """
+
+    def __init__(self, old_import, new_import):
+        super().__init__()
+        self.old_import = old_import
+        self.new_import = new_import
+
+    def __str__(self):
+        return (
+            "Please use import {self.new_import} instead of {self.old_import} as "
+            "'xblock-utils' package has been deprecated and migrated within 'xblock' package. "
```

Comment:
```suggestion
            "Please use import {self.new_import} instead of {self.old_import} because "
            "the 'xblock-utils' package has been deprecated and migrated to within 'xblock' package. "
```